xrdp/xrdp.ini

```diff
@@ -208,12 +208,6 @@ password=ask
 
 [rdp-any]
 name=rdp-any
-lib=librdp.so
-ip=ask
-port=ask3389
-
-[neutrinordp-any]
-name=neutrinordp-any
```

Comment:
Actually, I think all "*-any" sections should not be in the default install. They allow using xrdp as a proxy to another network, which it not an expected behavior. Instead, those sections should be in a sample file with extensive documentation.

I guess I'll just keep using `neutrinordp-any` for now, as this PR is about a different issue.